mud-contracts/world-v2/mud.config.ts

```diff
@@ -372,7 +303,6 @@ export default defineWorld({
           },
           key: ["smartObjectId"],
         },
-
         SmartGateLink: {
```

Comment:
Made couple of comments on the table names to keep it simple and intuitive. Let me know if it works